learn/cookbooks/digitalocean_droplet.md

```diff
@@ -137,11 +143,11 @@ curl -v https://<your-domain-name>/health
 The server should answer with a `200 OK` status code and the following body `{"status": "available"}` as shown in the example below:
 
 ```bash
-...
+…
 HTTP/1.1 200 OK
-...
+…
 {"status": "available"}
-...
+…
 ```
 
 **Enjoy**!
```

Comment:
Recently updated company policy dictates users should **not** be encouraged to enjoy themselves.

```suggestion

```